### PR TITLE
Editorial changes to READMEs

### PR DESCRIPTION
**Pull request by jmedved**
Please review the writeup on pulling images from Dockerhub in prod_vpp README. A similar writeup needs to be added to the docker/dev_vpp_agent/README. 

Also, you need to explain what kind of image is the Dockerhub image, since we have 4 options: dev env, shrinked dev env, prod, shrinked prod.

### CODE REVIEW

**Comment by lukasmacko** [comment]
It seems that you have deleted some files from vendor dir by mistake and it causes build failure.

File: README.md [hunk 1]
```diff
@@ -32,15 +32,16 @@ using [protobufs][3], which allow for the same functionality to be accessible
 over multiple transport protocols (HTTP, gRPC, Etcd, ...). Plugins use the 
 [GoVPP library][4] to interact with the VPP.
 
-The following figure shows a cloud-native VNF with its data plane based on
-VPP/DPDK and its management / control planes based on the VNF agent:
+The following figure shows the VN Agent in context of a cloud-native VNF,
```
Comment (lukasmacko) [inline README.md]: VNF agent?

File: README.md [hunk 2]
```diff
@@ -60,16 +61,22 @@ The set of plugins in the VPP Agent id as follows:
...
 
-* [agentctl](cmd/agentctl) - a CLI tool that shows the state of a set of VPP agents 
-   can configure the agents
-* [Docker based development environment](docker)
+* [agentctl](cmd/agentctl) - a CLI tool that shows the state of a set of 
+   VPP agents can configure the agents
+* [vpp-agent-ctl](vpp-agent-ctl/main.go) - a utility for testing VNF Agent 
+  configuration. It contains a set of pre-defined configurations that can 
+  be sent to the VPP Agent either interactively or in a script. 
+  predefined parts of configuration (aimed for testing purposes)
```
Comment (lukasmacko) [inline README.md]: this line should be removed

File: docker/dev_vpp_agent/README.md [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
+This image can be used to get started with the vpp-agent Go code. It 
+contains:
 
-- The development environment with all libs & dependencies required to build VPP / Go code,
-- A pre-built vpp ready to be used,
-- A pre-built Go agent.
+- The development environment with all libs & dependencies required 
+  to build both the VPP itself and the VPP Agent
+- A pre-built vpp ready to be used
+- A pre-built VPP Agent agent
```
Comment (lukasmacko) [inline docker/dev_vpp_agent/README.md]: A pre-built VPP agent

File: docker/dev_vpp_agent/README.md [hunk 4]
```diff
@@ -73,43 +84,45 @@ dev_vpp_agent_shrink                                  latest              bd2e76
...
 ```
 
 ### Running VPP and the Agent
 
-**NOTE: The Agent will terminate if it cannot connect to VPP and to a Etcd server. 
-If Kafka config is specified, a successful connection to Kafka is also required. 
-If Kafka config is not specified, the Agent will run without it, but all Kafka-related 
-functionality will be disabled.** 
+**NOTE: The Agent will terminate if it cannot connect to VPP and to a Etcd
+server. f Kafka config is specified, a successful connection to Kafka is
```
Comment (lukasmacko) [inline docker/dev_vpp_agent/README.md]: If

File: docker/dev_vpp_agent/README.md [hunk 5]
```diff
@@ -252,14 +278,21 @@ through these steps.
 
 - In Gogland, set `GOROOT` and `GOPATH` (`Preferences->Go->GOROOT`, `Preferences->Go->GOPATH`). Set `GOROOT` to where Go is installed (default: `/usr/local/go`) and the global `GOPATH` to the location of your Go home folder (`/Users/jmedved/Documents/Git/go-home/` in our example).
 
-- Create a new project in Gogland (`File->New->Project`, ) will popup the project creation window. Enter your newly created Go home folder (`/Users/jmedved/Documents/Git/go-home/` in our example) as the location and accept the default value for the SDK. Click `Create`, and you should now be able to browse the Agent source code in Gogland.
+- Create a new project in Gogland (`File->New->Project`, ) will popup the 
+  project creation window. Enter your newly created Go home folder 
+  (`/Users/jmedved/Documents/Git/go-home/` in our example) as the locatio
```
Comment (lukasmacko) [inline docker/dev_vpp_agent/README.md]: location (n is at a new line)

File: README.md [hunk 6]
```diff
@@ -60,16 +61,22 @@ The set of plugins in the VPP Agent id as follows:
 The VPP agent repository also contains tools for building and troubleshooting 
 of VNFs based on the VPP Agent:
 
-* [agentctl](cmd/agentctl) - a CLI tool that shows the state of a set of VPP agents 
-   can configure the agents
-* [Docker based development environment](docker)
+* [agentctl](cmd/agentctl) - a CLI tool that shows the state of a set of 
+   VPP agents can configure the agents
+* [vpp-agent-ctl](vpp-agent-ctl/main.go) - a utility for testing VNF Agent
```
Comment (lukasmacko) [inline README.md]: invalid link it should be `cmd/vpp-agent-ctl/main.go`

**Comment by jmedved** [comment]
I did not delete anything. Jmedved is (was yesterday evening) in sync with master. But it contains some uncommitted files in vendor which I can’t commit, since git says I do not have the permission to commit them. Any idea wtf can be the issue?


/Jan

**Pull request merged**